frontend/public/components/storage/create-pvc.tsx

```diff
@@ -301,7 +329,7 @@ export type CreatePVCFormProps = {
 
 export type CreatePVCFormState = {
   storageClass: string;
-  storageClassAccessMode: string;
+  storageClassAccessModes: string[];
```

Comment:
If you are removing storageClassAccessMode, you should remove it everywhere since it is no longer being used for anything meaningful